### ISSUE TITLE
New line character shortcut (Shift+Enter) not working on VR keyboards

### ISSUE THREAD

**Issue by skywindkitsune** [Neos-Metaverse/NeosPublic#2792]
### Describe the bug?

On Previous versions of Neos we were able to press Shift+Enter to create a new line character on Basic/Outline Text and now we are no longer able to do that.

### Relevant issues

No

### To Reproduce

1. Create a New Basic Text using the dev tool tip
2. Edit the text and attempt to create a new line by pressing shift + enter
3. it should just close the keyboard instead of creating a new line.

### Expected behavior

When pressing Shift+enter on the keyboard it should create a new line character not close the keyboard.

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2021.8.13.612

### What Platforms does this occur on?

Windows

### Did this work before?

Yes

### Reporters

Skywind Kitsune, Shadow Panther, Earthmark

**Comment by shadowpanther** [comment]
Looking at the recent changes, the only one related to keyboards was the yesterday's one:
```
- Fixed virtual shift triggering for other users in world-space virtual keyboards (based on report by @Earthmark)
-- Note there are more planned changes and fixes coming, but need certain things resolved around them first
```

**Comment by ukilop** [comment]
oh others are having this too? me, coffee, and 1 other in a work session experienced this as well, we chopped it up to complex world bug.

**Comment by Frooxius** [comment]
Should be fixed in 2021.8.14.443, thanks for report!
